plugins/org.eclipse.papyrus.uml.diagram.sequence.contribution/context/lightweight-seqd.contexts

```diff
@@ -0,0 +1,55 @@
+      <properties xsi:type="constraints:ValueProperty" xmi:id="_70idv6DJEeSZxfCXzZz3-w" name="umlClassName" value="Message"/>
+    </constraints>
+  </views>
+  <views xmi:id="_70idwKDJEeSZxfCXzZz3-w" elementMultiplicity="-1" name="MultipleMessage" sections="_70QJk6DJEeSZxfCXzZz3-w" automaticContext="true">
+    <constraints xsi:type="constraints:SimpleConstraint" xmi:id="_70idwaDJEeSZxfCXzZz3-w" name="isMultipleMessage">
+      <constraintType href="ppe:/environment/org.eclipse.papyrus.uml.properties/Model/Environment.xmi#//@constraintTypes.0"/>
+      <properties xsi:type="constraints:ValueProperty" xmi:id="_70idwqDJEeSZxfCXzZz3-w" name="umlClassName" value="Message"/>
+    </constraints>
+  </views>
+  <views xmi:id="_70idvaDJEeSZxfCXzZz3-l" name="SingleLifeline" sections="_70QJkqDJEeSZxfCXzZz3-l" automaticContext="true">
+    <constraints xsi:type="constraints:SimpleConstraint" xmi:id="_70idvqDJEeSZxfCXzZz3-l" name="isSingleLifeline">
+      <constraintType href="ppe:/environment/org.eclipse.papyrus.uml.properties/Model/Environment.xmi#//@constraintTypes.0"/>
+      <properties xsi:type="constraints:ValueProperty" xmi:id="_70idv6DJEeSZxfCXzZz3-l" name="umlClassName" value="Lifeline"/>
+    </constraints>
+  </views>
+  <dataContexts xmi:id="_700xgaDJEeSZxfCXzZz3-w" name="LWSequenceDiagram" label="Sequence Diagram">
+    <elements xmi:id="_700zkKDJEeSZxfCXzZz3-w" name="Message">
+      <properties xmi:id="_700xlKDJEeSZxfCXzZz3-w" name="name" description="The name of the NamedElement."/>
+      <properties xmi:id="_700zkaDJEeSZxfCXzZz3-w" name="messageKind" type="Enumeration" description="The derived kind of the Message (complete, lost, found or unknown)"/>
+      <properties xmi:id="_700zkqDJEeSZxfCXzZz3-w" name="messageSort" type="Enumeration" description="The sort of communication reflected by the Message"/>
+      <properties xmi:id="_700zk6DJEeSZxfCXzZz3-w" name="receiveEvent" type="Reference" description="References the Receiving of the Message"/>
+      <properties xmi:id="_700zlKDJEeSZxfCXzZz3-w" name="sendEvent" type="Reference" description="References the Sending of the Message."/>
+      <properties xmi:id="_700zlaDJEeSZxfCXzZz3-w" name="connector" type="Reference" description="The Connector on which this Message is sent."/>
+      <properties xmi:id="_700zlqDJEeSZxfCXzZz3-w" name="interaction" type="Reference" description="The enclosing Interaction owning the Message"/>
+      <properties xmi:id="_700zl6DJEeSZxfCXzZz3-w" name="argument" type="Reference" description="The arguments of the Message"/>
+      <properties xmi:id="_700zmKDJEeSZxfCXzZz3-w" name="signature" type="Reference" description="The definition of the type or signature of the Message (depending on its kind). The associated named element is derived from the message end that constitutes the sending or receiving message event. If both a sending event and a receiving message event are present, the signature is obtained from the sending event."/>
```

Comment:
That's a rather long description ...

Reply:
Yeah, probably doesn't make sense to have this long description here. This is the original description from Papyrus, but I'll shorten it.